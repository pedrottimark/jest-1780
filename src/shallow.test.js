import React from 'react';
import TestUtils from 'react-addons-test-utils';

import J from './shallow';

describe('J as a stick figure', () => {

  const x1 =  1 /  2;
  const y1 = 29 / 63;
  const x2 =  1 /  4;
  const y2 =  7 /  8;
  const j = <J x1={x1} y1={y1} x2={x2} y2={y2} stroke="#f7df1e" strokeWidth={6 / 63} />;

  it('consists of LineV and LineH components', () => {
    const shallowRenderer = TestUtils.createRenderer();
    const rendered = shallowRenderer.render(j);
    const output = shallowRenderer.getRenderOutput();
    expect(output).toMatchSnapshot();
  });

});
