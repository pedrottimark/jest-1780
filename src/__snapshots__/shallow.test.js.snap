exports[`J as a stick figure consists of LineV and LineH components 1`] = `
<g
  stroke="#f7df1e"
  strokeWidth={0.09523809523809523}>
  <LineV
    x={0.5}
    y1={0.4603174603174603}
    y2={0.875} />
  <LineH
    x1={0.5}
    x2={0.25}
    y={0.875} />
</g>
`;
